src/jobs/get-br-dependencies.yml

```diff
@@ -0,0 +1,46 @@
+parameters:
+  exec:
+    type: executor
+  resource-class:
+    type: string
+    default: medium
+  hex-validate:
+    type: boolean
+    default: true
+  env-setup:
+    type: steps
+    default: []
+  save-dl-cache:
+    type: boolean
+    default: true
+executor: << parameters.exec >>
+resource_class: << parameters.resource-class >>
+steps:
+  - checkout
+  - steps: << parameters.env-setup >>
+  - install-elixir:
+      version: $ELIXIR_VERSION
+  - install-hex-rebar
+  - install-nerves-bootstrap
+  - mix-deps-get
+  - when:
+      condition: << parameters.hex-validate >>
+      steps:
+        - run:
+            name: Validate Hex package
+            command: mix hex.build
+  - restore-nerves-downloads
+  - run:
+      name: Get Buildroot dependencies
+      command: |
+        ./deps/nerves_system_br/create-build.sh nerves_defconfig ~/temp_nerves_system
```

Comment:
This feels a bit hacky, but I was looking for a way to avoid the system automatically compiling when calling `mix` so that I could just pull the Buildroot dependencies. If there's a way to get the Nerves tooling to create the Buildroot project in `<project>/.nerves/artifacts/<nerves_system>` without compiling the system, that may be better.